docs/cloud/managing-airbyte-cloud.md

```diff
@@ -259,6 +287,23 @@ To display **Connection State**:
 
     The **Connection State** displays. 
 
+## Choose the data residency for a connection
+To choose the data residency for an existing connection: 
+
+1. On the [Airbyte Cloud](http://cloud.airbyte.io) dashboard, click **Connections** in the navigation bar and then click the connection that you want to change. 
+
+    The Connection page displays. 
+
+2. Click the **Settings** tab. 
+
+3. Click the **Data residency** dropdown and choose the location for your default data residency.
+
+:::note 
+    
+You can also choose data residency when creating a [new connection](https://docs.airbyte.com/cloud/getting-started-with-airbyte-cloud#set-up-a-connection), or you can set the [default data residency](#choose-your-default-data-residency) for all of your connections. 
```

Comment:
```suggestion
You can also choose data residency when creating a [new connection](https://docs.airbyte.com/cloud/getting-started-with-airbyte-cloud#set-up-a-connection), or you can set the [default data residency](#choose-your-default-data-residency) for your workspace.
```

Just a suggestion: this could be interpreted such that changing your default data residency affects all of your connections, which it does not.

Reply:
Good catch! I'll edit this in another PR